product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -15,7 +15,7 @@ Depending on where your older and newer versioned BigAnimal instances are locate
 
 ### Create a new BigAnimal instance
 
-Migrating between major versions first requires creating a new BigAnimal instance with the newer version of Postgres. This procedure does not work for upgrading a distributed high-availability BigAnimal instance.
+Migrating between major versions first requires creating a new BigAnimal instance with the newer version of Postgres. This procedure does not work for upgrading from a single node or standard high availability cluster to a distributed high-availability BigAnimal instance.
```

Comment:
The second sentence here (starting with "This procedure") is rather a non-sequitur anyway. Shouldn't I know that this process is limited to a certain class of instances _before_ I get to the "create a cluster" step?